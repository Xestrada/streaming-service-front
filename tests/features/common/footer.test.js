import React from 'react';
import  Footer  from '../../../src/features/common';


describe('common/footer', () => {
  it('renders correctly', () => {
    const renderedComponent = <Footer />;
    expect(renderedComponent).toMatchSnapshot();
  });
});
