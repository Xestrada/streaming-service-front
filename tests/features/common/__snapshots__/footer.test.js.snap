// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/footer renders correctly 1`] = `<Unknown />`;
